lib/ansiblelint/utils.py

```diff
@@ -52,6 +53,27 @@
 
 PLAYBOOK_DIR = os.environ.get('ANSIBLE_PLAYBOOK_DIR', None)
 
+INVALID_CONFIG_RC = 2
+ANSIBLE_FAILURE_RC = 3
+
+VERBOSITY_MAP = {
+    0: logging.NOTSET,
+    1: logging.INFO,
+    2: logging.DEBUG
+}
+
+logger = logging.getLogger("ansiblelint")
+handler = logging.StreamHandler()
+formatter = logging.Formatter('%(levelname)-8s %(message)s')
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+logger.setLevel(logging.DEBUG)
```

Comment:
I think it's not a good idea to have this on the module level globally. Import-time evaluation is usually not a good thing. I'd prefer a setup function called when the app starts.

Reply:
@webknjaz Can you please give an existing logging implementation as an reference example so I can  follow that? 

I understand what you mean and I agree it may not be the best implementation but I also want to avoid rewriting this PR forever. For example, I would have avoided implementing set_logging_verbosity_level and create an init_logging() function instead if that remark was added earlier.

Another approach is to merge it in its current form and implement the other changes as a follow-up, as the logging calls would not change.

Reply:
Basically, I want some `initialize_logger()` implemented here and `main()` calling it at the very beginning, before anything else. I understand that you want to merge incrementally so I won't block this PR if you postpone this specific change.